### PR TITLE
Update PPP - Hao Wei

### PR DESCRIPTION
**Pull request by KwanHW**
# Changes
This PR amends the PPP by adding recent contributions to the program as well as standardizing the program description.

# Related Issues
None.

### CODE REVIEW

File: docs/team/kwanhw.md [hunk 1]
```diff
@@ -20,15 +20,17 @@ Given below are my contributions to the project.
   * Justification: This improves the usability of the application, as it provides more options for the user to search for contacts.
 
 * **Code contributed**:
-  * 2000+ LoC contributed to the project
+  * 2800+ LoC contributed to the project
```
Comment (sharpstorm) [inline docs/team/kwanhw.md:23]: Add a link to reposense, it is a requirement according to the PPP page.

File: docs/team/kwanhw.md [hunk 2]
```diff
@@ -20,15 +20,17 @@ Given below are my contributions to the project.
...
   * Test cases for the tags subsystem
 
 * **Project management**:
+    * Reviewed Pull Requests
```
Comment (sharpstorm) [inline docs/team/kwanhw.md:27]: Duplicate of the point below.

File: docs/team/kwanhw.md [hunk 3]
```diff
@@ -47,7 +49,5 @@ Given below are my contributions to the project.
         * Added use cases for tags
 
 * **Community**:
-    * `to be added soon`
-
-* **Tools**:
-    * `to be added soon`
+    * Reviewed PRs within the team ([20+ comments](https://nus-cs2103-ay2122s2.github.io/dashboards/contents/tp-comments.html) across [50+ PRs](https://github.com/AY2122S2-CS2103-W17-1/tp/pulls?q=is%3Apr++is%3Aclosed+reviewed-by%3Akwanhw))
```
Comment (sharpstorm) [inline docs/team/kwanhw.md:52]: The number here don't look correct.

**Review by sharpstorm — COMMENTED** [review]
You missed the part on adding tabbed UI and the associated styling.

File: docs/team/kwanhw.md (hunk 1 above)
Comment (KwanHW) [reply docs/team/kwanhw.md]: Added in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/322/commits/9acfd793b792ca9c3e94edf65a30df349281b70d

File: docs/team/kwanhw.md (hunk 2 above)
Comment (KwanHW) [reply docs/team/kwanhw.md]: Thanks for the catch! Was an error from resolving the merge conflict. Fixed in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/322/commits/9acfd793b792ca9c3e94edf65a30df349281b70d

File: docs/team/kwanhw.md (hunk 3 above)
Comment (KwanHW) [reply docs/team/kwanhw.md]: Amended in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/322/commits/9acfd793b792ca9c3e94edf65a30df349281b70d

**Review by sharpstorm — APPROVED** [review]
LGTM.

**Review by dandaandaaaaaan — APPROVED** [review]
lgtm

**Pull request merged**